.github/CODEOWNERS

```diff
@@ -93,3 +93,24 @@ app/scripts/controllers/swaps                         @MetaMask/swaps-engineers
 shared/constants/permissions.ts      @MetaMask/snaps-devs
 ui/helpers/utils/permission.js       @MetaMask/snaps-devs
 ui/hooks/useTransactionInsights.js   @MetaMask/snaps-devs
+
+# Wallet UX
+ui/components/multichain              @MetaMask/metamask-wallet-ux
+ui/components/app/whats-new-popup     @MetaMask/metamask-wallet-ux
+ui/css                                @MetaMask/metamask-wallet-ux
+ui/pages/home                         @MetaMask/metamask-wallet-ux
+ui/pages/onboarding-flow              @MetaMask/metamask-wallet-ux
+
+# Assets
+ui/components/app/add-network                         @MetaMask/metamask-assets
+ui/components/app/auto-detect-nft                     @MetaMask/metamask-assets
+ui/components/app/auto-detect-token                   @MetaMask/metamask-assets
+ui/components/app/import-token                        @MetaMask/metamask-assets
+ui/components/app/nft-default-image                   @MetaMask/metamask-assets
+ui/components/app/nft-details                         @MetaMask/metamask-assets
+ui/components/app/nft-options                         @MetaMask/metamask-assets
+ui/components/app/nfts-detection-notice-import-nfts   @MetaMask/metamask-assets
+ui/components/app/nfts-items                          @MetaMask/metamask-assets
+ui/components/app/nfts-tab                            @MetaMask/metamask-assets
+ui/components/ui/deprecated-networks                  @MetaMask/metamask-assets
+ui/components/ui/nft-collection-image                 @MetaMask/metamask-assets
```

Comment:
it seems to think this file has errors

<img width="753" alt="Screenshot 2024-08-19 at 11 22 38 AM" src="https://github.com/user-attachments/assets/aa639646-e38f-485c-990a-7d519eeb4086">

Reply:
- https://github.com/orgs/MetaMask/teams/metamask-assets
- https://github.com/orgs/MetaMask/teams/wallet-ux

Those exist ^.  So it might need to be `@MetaMask/wallet-ux` ?

Not sure about the other conditions like `has write access`

Reply:
Still says it has errors: https://github.com/MetaMask/metamask-extension/blob/codeowners/.github/CODEOWNERS